packages/scss/src/components/verticalNavigation/index.scss

```
@@ -3,6 +3,10 @@
 .verticalNavigation {
 	@include vars;
 	@include component;
+
+	&.mod-iconless {
```

```suggestion
	&.mod-iconLess {
```

we are not in camelCase here?

---

It's used as a suffix imho 🙂

---

`.mod-noIcon` might be easier to understand?

---

Agree, but we already have iconless references in a couple of other components so it would break consistency.